__tests__/commands/install/resolutions.js

```diff
@@ -51,3 +51,25 @@ test.concurrent('should warn when resolution entries are incorrrect or incompati
   expect(error).toContain('Resolution field "wrongversion" has an invalid version entry and may be ignored');
   expect(error).toContain('Resolution field "invalidname/" does not end with a valid package name and will be ignored');
 });
+
+test.concurrent('install with resolutions should correctly install simple scoped packages', (): Promise<void> => {
+  return runInstall({}, {source: 'resolutions', cwd: 'scoped-simple'}, async config => {
+    expect(await getPackageVersion(config, '@scoped/a')).toEqual('1.0.0');
+    expect(await getPackageVersion(config, '@scoped/b')).toEqual('2.0.0');
+  });
+});
+
+test.concurrent('install with resolutions should correctly install toplevel scoped packages', (): Promise<void> => {
+  return runInstall({}, {source: 'resolutions', cwd: 'scoped-toplevel'}, async config => {
+    expect(await getPackageVersion(config, '@scoped/a')).toEqual('1.0.0');
+    expect(await getPackageVersion(config, '@scoped/b')).toEqual('2.0.0');
+  });
+});
+
+test.skip('install with resolutions should correctly install nested scoped packages', (): Promise<void> => {
+  return runInstall({}, {source: 'resolutions', cwd: 'scoped-nested'}, async config => {
+    expect(await getPackageVersion(config, '@scoped/a')).toEqual('1.0.0');
+    expect(await getPackageVersion(config, '@scoped/b')).toEqual('1.0.0');
+    expect(await getPackageVersion(config, '@scoped/a/@scoped/b')).toEqual('2.0.0');
+  });
+});
```

Comment:
nit, would you mind just deleting this test case and the related files then add them back later, in case you don't get to the issue? It also makes it easier to review in the pr.